d3ledger/back-office
# D3-804 | Implement limits

# Description
There is typo in branch name
https://soramitsu.atlassian.net/browse/D3-804

#### Works done
- [x] Add new limit
- [x] Edit limit
- [x] Delete limit
- [x] Open \ Close tabs
- [x] Created new component for limit card

# How to test
- `yarn` to install all dependencies
- `yarn serve` to run application
- Go to settings page and check new limit feature
